crates/cli/src/config.rs

```diff
@@ -11,3 +12,8 @@ pub struct CliConfig {
 	#[clap(long, env = "HASH_COST", default_value = "12")]
 	pub password_hash_cost: u32,
 }
+
+pub struct BundledConfigs {
```

Comment:
I'm not 100% sure we need this struct. The `StumpConfig` has both of the values present in `CliConfig`, so you could probably just get away with just using `StumpConfig`? Renaming `hash_cost` to `password_hash_cost`

Reply:
Great point - I changed things up a bit to remove the surplus struct. Give it a glance and see what you think.